sphinx_gallery/tests/test_gen_gallery.py

```diff
@@ -166,6 +169,23 @@ def test_config_backreferences(config_app):
     assert build_warn == ""
 
 
+@pytest.mark.skipif(sys.version_info[:2] == (3, 4),
+                    reason="Won't work on Python 3.4")
```

Comment:
I'm concerned that this won't work on py3.4, Why?

Reply:
See discussion above

https://github.com/sphinx-gallery/sphinx-gallery/pull/244#issuecomment-345439680